### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by AnicetFantomas**
In this pull request we performed :

- Use basic JavaScript syntax.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Understand how to parse a Figma design to create a UI.
- Use objects to store and access data.

### CODE REVIEW

File: pop.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+
+.main-for-pop-up {
+  background-color: rgba(0, 0, 255, 0.438);
```
Comment (BenMukebo) [inline pop.css:3]: - [ ] Your JS syntax is correct, keep up the great job, but I think there are a few things you will need to work on the styling, this gif below compared to the image design, their pop-up container (over-lay is not meant to have that purple color, instead it should be light gray with some transparency, please refer to the Figma design for `color, padding, margin, font-family, and  font-size
- [ Your Desin]
- 
anicet
 

- [ ] To change
- your top section ( canopy . backend ...) is not aligned and does not match the design, please fix that.
- your closing button is fixed but when the user scrolls the pop-up window, as you can see from the gif, it gets confused with the text and images which is not pretty.
- In your paragraph, please select the correct color position, font size, font-weight, and font family as per the Figma design
- Your technologies, please align and style them as per the Figma design below

- [LET'S TAKE A LOOK AT THE FIGMA]
-  MOBILE (470)

mob pop

-  DESKTOP (992)
pop desk

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi ** @AnicetFantomas, **

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [ ] Kindly, Add a descriptive README file to your project as required please use this [template](https://github.com/microverseinc/readme-template) You can copy past but remember to customize it to your project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: pop.css (hunk 1 above)
Comment (AnicetFantomas) [reply pop.css:3]: Thank you for your feedback @BenMukebo 

You are really doing a great job 💯 

Here bellow in my next commits that I took time to respect each one of your requested changes. 
nevertheless one point stay unsolved, When confuguring the close button, it acting on my js syntax,
Actually; You will see that I've put an other one on each pop-up window and I really tried over and over again ! But I stay open to any help

Hope that you'll understand !!

Thank you again and happy coding !!

File: index.html [hunk 2]
```diff
@@ -79,126 +64,21 @@ <h1>I'm Anicet<br />Glad to see <br />you!</h1>
...
-                        <li id="css"><a href="#">css</a></li>
-                        <li id="javaScript"><a href="#">javaScript</a></li>
-                    </ul>
-                    <div class="to-project"><a href="#">See Project</a></div>
-                </div>
-            </div>
+        <div class="work-section" id="ancre1">
+        </div>
+        <div class="popup-container display-none">
+            <img id="closeBtn" src="images/exitIcon.png" alt="This button will close the pop-up window">
```
Comment (AnselemOdims) [inline index.html:71]: - [ ] As already suggested by the previous reviewer, kindly make sure the close button for the modals is given a `position: fixed` property. You can remove the previous button image and put the event listener in the `x` instead.
- [ ] Also as suggested, kindly make sure the modal paragraph follows the correct color position, font size, font-weight, and font family as per the Figma design
- [ ] Also the project topic(in this case `CANOPY`) seems to be flowing out of the modal. Please fixt that also

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @AnicetFantomas , 
Good job implementing some changes suggested by the previous reviewer, but you still have a bit to go.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @AnicetFantomas ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

#### - [ ] ✅  I salute your efforts, you have tried as better as you could, keep that great work Anicet 💪🏻 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: pop.css (hunk 1 above)
Comment (BenMukebo) [reply pop.css:3]: Hello @AnicetFantomas,
 Great work, am proud of your accomplishment 😄, congrats 💯

**Pull request merged**